lib/model/Nutriments.dart

```diff
@@ -1513,4 +1514,179 @@ class Nutriments extends JsonObject {
     'omega-6-fat',
     'trans-fat',
   };
+
+  bool get hasData =>
```

Comment:
I'm not convinced by that bug-proned method. I'll detail my thoughts a bit later.